crane/data.py

```diff
@@ -173,14 +173,16 @@ def load_all(app):
                  for f in fnmatch.filter(files, '*.json')]
         # load data from each file
         for metadata_file_path in paths:
-            repo_id, repo_tuple, image_ids = load_from_file(metadata_file_path)
-            if isinstance(repo_tuple, V1Repo):
-                v1_repos[repo_id] = repo_tuple
-                for image_id in image_ids:
-                    images.setdefault(image_id, set()).add(repo_id)
-            else:
-                v2_repos[repo_id] = repo_tuple
-
+            try:
```

Comment:
I suggest to apply try/except just for the load of the metadata file itself.
It is a good practice to put into the 'try' clause less code possible and just the line/lines of code which can actually trigger the exception. This way it is easier to troubleshoot. 

```
for metadata_file_path in paths:
            try:
                repo_id, repo_tuple, image_ids = load_from_file(metadata_file_path)
            except Exception, e:
                logger.error('Could not load metadata for: %s. Skipping %s' % (metadata_file_path, str(e)))
                continue
            ....
```

Reply:
Adding try for just loading would introduce more intermediate variables and I wanted to avoid that. I ran nosetest coverage and it did not show any coverage issues. I will try to add tests. However, since, we are only logging, I am not sure how much test is going to  add to the coverage.